### PR TITLE
Loop mi model changes 2

### PR DESCRIPTION
**Pull request by sujit22993**
Saving Textlocal sms id in another table

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Abhishek-lodha) [inline .vscode/settings.json]: remove this file

File: loop_ivr/helper_function.py [hunk 2]
```diff
@@ -128,6 +130,11 @@ def send_sms_using_textlocal(user_no, sms_body, price_info_incoming_obj):
             else:
                 price_info_incoming_obj.textlocal_sms_id += ',' + message_id
             price_info_incoming_obj.save()
+            print 'PIIOBJ saved!'
```
Comment (Abhishek-lodha) [inline loop_ivr/helper_function.py]: remove print statements

File: loop_ivr/models.py [hunk 3]
```diff
@@ -72,3 +72,13 @@ class SubscriptionLog(LoopModel):
 
     def __unicode__(self):
         return "%s (%s)" % (self.subscription, self.status)
+
+
+class SmsStatus(LoopModel):
+    id = models.AutoField(primary_key=True)
+    price_info_incoming_id = models.ForeignKey(PriceInfoIncoming)
```
Comment (Abhishek-lodha) [inline loop_ivr/models.py]: rename column . remove id from column name

File: loop_ivr/helper_function.py [hunk 4]
```diff
@@ -128,6 +130,11 @@ def send_sms_using_textlocal(user_no, sms_body, price_info_incoming_obj):
             else:
                 price_info_incoming_obj.textlocal_sms_id += ',' + message_id
```
Comment (Abhishek-lodha) [inline loop_ivr/helper_function.py]: stop entering data into PII textlocalsmsid column

Comment (sujit22993) [reply loop_ivr/helper_function.py]: Done

File: loop_ivr/models.py (hunk 3 above)
Comment (sujit22993) [reply loop_ivr/models.py]: done

File: loop_ivr/helper_function.py (hunk 2 above)
Comment (sujit22993) [reply loop_ivr/helper_function.py]: Done

File: .vscode/settings.json (hunk 1 above)
Comment (sujit22993) [reply .vscode/settings.json]: Done

File: loop_ivr/management/commands/textlocal_sms_status.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        params = {'apikey': apikey, 'message_id': messageID}
+        f = urllib.urlopen('https://api.textlocal.in/status_message/?'
+            + urllib.urlencode(params))
+        return (f.read(), f.code)
+
+    def handle(self, *args, **options):
+        today = datetime.now().date()
+        sms_status_obj = SmsStatus.objects.filter(api_call_initiation_time__contains=today)
+        for smsobj in sms_status_obj:
+            resp, code = self.get_textlocal_sms_status(TEXTLOCAL_API_KEY, smsobj.textlocal_sms_id)
```
[the remark this answers is not in the document]
Comment (sujit22993) [reply loop_ivr/management/commands/textlocal_sms_status.py]: removed

File: loop_ivr/management/commands/textlocal_sms_status.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import urllib
+from loop_ivr.models import SmsStatus
+from datetime import datetime, timedelta
```
[the remark this answers is not in the document]
Comment (sujit22993) [reply loop_ivr/management/commands/textlocal_sms_status.py]: removed

**Pull request merged**